src/pages/library/Code.js

```diff
@@ -122,10 +134,15 @@ const Code = () => {
           div({ style: styles.header }, 'GATK4 Best Practices workflows'),
           div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
             _.map(method => {
-              const { namespace, name } = method
+              const { namespace, name, id } = method
+              const isMethodsRepoMethod = namespace && name
```

Comment:
For explicitness and consistency with the codebase (at least the more recent portion):
```suggestion
              const isMethodsRepoMethod = !!namespace && !!name
```